src/MessagePack.UnityClient/Assets/Scripts/MessagePack/MessagePackWriter.cs

```diff
@@ -1210,7 +1214,7 @@ private static void WriteBigEndian(ulong value, Span<byte> span)
             // solves heuristic length check
 
             // ensure buffer by MaxByteCount(faster than GetByteCount)
```

Comment:
this comment is now stale if you remove the use of `GetMaxByteCount`.

Reply:
Thank you!